### ISSUE TITLE
Use options of VLC

### ISSUE THREAD

**Issue by brandy253** [mzafers/QtVlcMediaPlayer#2]
Thanks for your lib.
Btw, in libvlc there is a function called libvlc_media_add_option()
Is there anyway to use that function in this library?

**Comment by mzafers** [comment]
Ok, I'll add it with new libvlc api.

**Comment by brandy253** [comment]
Thank you fo your greate work.

**Comment by Psykomusic** [comment]
Hi ! 
Thx for you work it's very good and I succes to build my app with Qt 5.9 and Android 23  ! 
I post here because I want to add vlc-option but I can't find where declare it. Can you show us or add some documentation ? 
In fact i want to set the option --network-caching.
Thx Again

**Comment by mzafers** [comment]
Hi. I couldn't add libvlc options sorry. If you want to change network caching value you can find it in QtMediaPlayer.java file, getLibOptions function. Then you must compile QtMultimedia module again. Not need to compile all QtMultimedia, just compile jar files. It generates 2 jar files.

**Comment by dastious** [comment]
Any help on how to compile the jar files ? I tried to update to libvlc3 without success. I tried to compile the whole qt lib (5.7.1) for android but i get some error.
There is package org.videolan.libvlc the qtmultimedia java files, but i don't understand where do i need to put the libvlc android files or folder ?

I can publish the new one, but i'm a bit lost for now.

**Comment by mzafers** [comment]
Hi. libvlc3 is a little bit different as far as I know. You cannot compile the project only copying new files. You should check vlc3 java files and change some files in project. In fact, I want to update project to Qt 5.10 and libvlc3. But I couldn't yet, no time.

**Comment by dastious** [comment]
Yes i know, but my question was do you know how to compile qtmultimedia or just the jar files without compiling all the qt modules ? (which is 2hours of compilation)

**Comment by mzafers** [comment]
No need to compile all qtmultimedia. You can compile jar subdir only. It takes 10 seconds max.

**Comment by mzafers** [comment]
Just compile qtmultimedia/src/plugins/android/jar/ subproject.
